docs/src/inputs/radio/examples/radioSamplePill.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Radio } from 'react-cm-ui';
+
+function RadioSample() {
+    return (
+        <React.Fragment>
+            <Radio
+                checked={0}
+                pill
+                onChange={() => false}
+            >
+                <Radio.Item label="Selected" />
+                <Radio.Item label="Unselected" />
+                <Radio.Item label="Unselected" />
+            </Radio>
+
+            <Radio
+                checked={0}
+                pill
+                onChange={() => false}
+                disable
+            >
+                <Radio.Item label="Disabled Selected" />
+                <Radio.Item label="Disabled Unselected" />
+                <Radio.Item label="Disabled Unselected" />
+            </Radio>
+        </React.Fragment>
```

Comment:
Please put in a Grid. Example can be followed here: `./docs/src/inputs/button/examples/exampleButtonSolid.jsx`

Reply:
done!